app/routes/games/list.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  actions: {
+    newGame: function () {
+      // The user must not be anonymous.
+      if (this.get('session.user.isAnonymous')) {
```

Comment:
:+1: To accomplish this and to make dealing with a user's decks easier, I'm making a `hasMany('deck')` on the user model. Do you think that makes sense? I'll have some commits to push in a minute to show you what it looks like.